src/spatiotemporalsystem.jl

```diff
@@ -54,7 +54,7 @@ function DynamicalSystemsBase.trajectory(ds::STS{Φ, T, F, P}, t, u = ds.prob.u0
     ti = ds.prob.t0
     tvec = ti:dt:t
     L = length(tvec)
-    data = Vector{Array{T, Φ}}(L)
+    data = Vector{Array{T, Φ}}(undef,L)
```

Comment:
Do we still want this `SpatioTemporalSystem`? If yes, then I should rework it to fit the new dynamical systems framework.